### DISCUSSION TITLE
When will a next release be planned?

### DISCUSSION THREAD

**Discussion by Marvin-Brouwer** [Huachao/vscode-restclient#770] [Q&A]
Hi,

I'm pretty happy with this extension, it has helped me a lot.
However, I'm finding myself in need of a couple of changes that are either in progress or fixed but not released yet.
And because of that, I'm kind of curious if there are any plans on releasing a new version in the foreseeable future?

I looked in the milestones but I can only find a backlog.

**Comment by medteck** [comment]
Same here! There's some merged features I'd love to have but they're not released.
